docs/spec/nmt.md

```diff
@@ -167,16 +168,89 @@ This is the same as the queried range.
 - In the event that the queried range is outside the range of the NMT leaves indices, an empty `proof` is returned.
 This empty `proof` consists of an empty set of `nodes` and an empty range, where `start` is `0` and `end` is also `0`.
 
-#### Index-based Merkle Range Proof Verification (for leaves with identical namespace IDs)
+#### Index-based Merkle Range Proof Verification (for leaves with identical namespaces)
 
 Let `leaves` refer to the namespaced messages of the leaves in the queried range `[start, end)`, while `T` represents the root of the tree against which the `proof` is being verified.
 The `proof` can be verified by taking the following steps:
 
-- Verify that all the leaves are namespaced messages with the same namespace ID.
+- Verify that all the leaves are namespaced messages with the same namespace.
 - Compute the tree root `T'` using the leaves and the `proof.nodes`.
 If the computed root `T'` is equal to the expected root `T` of the tree, then the `proof` is valid.
 To compute the tree root `T'`, the [namespaced hash function](#namespaced-hash) should be utilized.
 
+### Short Namespace Absence Proof
+
+The short namespace absence proof is a more efficient variant of the regular namespace absence proof.
+It differs from the original namespace absence proof definition  where instead of providing the inclusion proof of the `LeafHash` to the root (as `proof.nodes`),
```

Comment:
```suggestion
It differs from the original namespace absence proof definition where instead of providing the inclusion proof of the `LeafHash` to the root (as `proof.nodes`),
```

Reply:
Addressed in https://github.com/celestiaorg/nmt/pull/209/commits/5946b91c6735ad609229942334882505f3e3ad79